src/reducers/app/thunks.ts

```diff
@@ -534,7 +534,7 @@ export const shareReport = () => async (dispatch: any, state: any) => {
   const serializedQuery = JSON.stringify(query);
   const hostUrl = window.location.origin;
   const encodedUrl = base64url(serializedQuery);
-  const shareLink = `${hostUrl}?e=${encodeURIComponent(selectedConfig.displayName)}/${encodeURIComponent(selectedEntity)}&q=${encodedUrl}`;
+  const shareLink = `${hostUrl}/${selectedConfig.platform}/${selectedConfig.network}/${selectedEntity}/query/${encodedUrl}`;
```

Comment:
This is problematic because it's not backwards compatible

Reply:
Do you want support previous query? 
/?e=Tezos%20Mainnet/accounts&q=:queryID

Reply:
yes